_plugins/update_author_data.rb

```diff
@@ -4,14 +4,15 @@
 # Detects if authors have published posts
 # And renders a page for them accordingly
 # at /author/:name/
-Jekyll::Hooks.register :site, :after_init do
```

Comment:
If we're not using this plugin any more, let's remove it.

(as long as we understand what it was doing)